interview-questions.md

```diff
@@ -8,48 +8,48 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: JSX from my understanding allows html syntax to function just like javascript. In order to use JSX we need to use curly braces.
 
-Researched answer:
+Researched answer: JSX is an extnesion for javascript that lets us use html syntax in javascript code. JSX is usually used in react. In JSX we can use the html elements we are familiar with, like div, and h1 in Javascript. One of the benefits of using JSX is that with it we can write code that is easier to debug.
 
 1. What is yarn? What file(s) are modified when you run the command yarn in your terminal?
 
-Your answer:
+Your answer: yarn is a package manager that collects all the necessary files we need to create a react app. It is also the command used in the terminal to run a test with jest. I'm not 100% sure what files are modified when we run yarn but i imagine it has something to do with the package files that show up in vscode after we reate a react app;
 
-Researched answer:
+Researched answer:Yarn is an acronym that stands for yet another resource negotiator. It is a package manager that all the code snippets and dependencies necessary for our code. The filed yarn.lock, package.json, node_modules, .yarnc, and .yarn-integrity are all modified when we run yarn in the terminal. Thses are the most common files modified by yarn when ran in the terminal.
 
 3. What is a React component?
 
-Your answer:
+Your answer: A react component is any element of a react app that is interactive on the webpage. For example, if we want to create a navigation bar that the user can click on to get directions, we will need to create a component in our text editor to store that logic.
 
-Researched answer:
+Researched answer: A react Component is a reusable building block for user interface elements. Think of a component as a javascript function that returns a JSX element. Components are either functional components or class based components. Components let us create the UI so that it is easier to test and build.
```

Comment:
Awesome job mentioning functional and class based components. Functional are more popular but you could also run into class components as well.